### ISSUE TITLE
Distance Setup Project Wizard Internal Error

### ISSUE THREAD

**Issue by alexandra-ornitho79** [DistanceDevelopment/distance-bugs#186]
Hello,
I downloaded Distance 7.3 and was using it without problems. But since a few days, I have this message that appears each time I try to create a new project. If I click on "Ok" and continue to import my data, then the analysis can't run..
First I tried to download the software on another computer and it worked, but today I have the same error message.
I also tried to uninstall and reinstall the software but nothing changed.
If you know what's wrong and how I can fix it I would be very grateful.
Thanks
Alexandra
![Capture](https://user-images.githubusercontent.com/49938745/56643985-14f32380-667b-11e9-9559-845d1a6d5dba.PNG)

**Comment by erex** [comment]
Alexandra

We've not heard about a problem such as this.  So no answers at this time, simply more questions. Can you describe the events that lead up to receiving the error message?  Do I have this correct:

- The error appears when you click the "Next" button on Step 1 of the New Project Setup Wizard
- What occurred the previous time you used Distance?  
- You had previously imported data into Distance 7.3, or you opened an existing project (from a previous version of Distance) into Distance 7.3?

**Comment by alexandra-ornitho79** [comment]
Yes this message appears right after I click on the "Next" button, and before I even try to import data.
The previous time there was no error message,I could import my data and start the analysis without any problem.
Since I started using the software, it's with Distance 7.3 so no change with the version. 
Each time I try to create a new project and import new data, it worked in the beginning and now I have this message.

**Comment by erex** [comment]
Thanks Alexandra, this is useful.  I don't know how we can replicate your problem given it does not seem to be related to analysis.  Usually when people have problems using Distance we say, send us your project so we can investigate.  However, as the problem arises before you can create a project, there is nothing for you to send.

Are you able to open the project which you successfully completed?  If so, use the File | Export command from the menu to create a .zip file.  Then place that into a Github response; perhaps some clues lie there.

**Comment by lenthomas** [comment]
Thanks both - this is quite mysterious.  We've never seen this before, and it's going to be hard to diagnose unless we can reproduce it.  

I agree with Eric's comment about giving us a copy of the project - that might help us.

Also, can you tell me what happens if you choose the other options on that Wizard page before you press next.  In particular the last option, to Exit the wizard - do things work then?  If that works, try all the other options and let us know what you get.

Lastly, please tell us your operating system and any regional settings -- e.g., on my computer the Region  Settings page in Windows 10 is "United Kingdom" and Regional Format is "Recommended [English (United Kingdom)]".

Thanks!

**Comment by LHMarshall** [comment]
Hi Alexandra,
I've done a bit of reading and the issue seems to generally be related to combo box text properties becoming read only although I'm not sure why it is happened on your particular machine (but there are some combo boxes on the page it is trying to load). If this is the case then other options on that page should work as per Len's suggestion, so that would be good to verify.

I'll do some more digging and see if I can come up with a suggestion of how to fix it.
Do you know if your machine has run any updates recently? Knowing about your windows version would also be helpful. 

Thanks, Laura

Dr. L. Marshall Centre for Research into Ecological and Environmental Modelling University of St. Andrews 

    On Wednesday, 24 April 2019, 10:44:15 BST, Len Thomas <notifications@github.com> wrote:  
 
 
Thanks both - this is quite mysterious. We've never seen this before, and it's going to be hard to diagnose unless we can reproduce it.

I agree with Eric's comment about giving us a copy of the project - that might help us.

Also, can you tell me what happens if you choose the other options on that Wizard page before you press next. In particular the last option, to Exit the wizard - do things work then? If that works, try all the other options and let us know what you get.

Lastly, please tell us your operating system and any regional settings -- e.g., on my computer the Region Settings page in Windows 10 is "United Kingdom" and Regional Format is "Recommended [English (United Kingdom)]".

Thanks!

**Comment by LHMarshall** [comment]
Also Alexandra, when you say you contiue and can import the data but the analysis won't run what do you see at this point? What error messages are you given?

**Comment by LHMarshall** [comment]
Hi Alexandra, I just wondered if you had tried the other project creation options? Or have you had any luck getting Distance to work for you? I think I have one suggestion but would like to hear some more information from you first, best wishes, Laura

**Comment by erex** [comment]
@alexandra-ornitho79 

Alex

Are you the same ornithologist with whom I just had the Zoom consultation about the invisible results window content?  I wonder if the problems are related?
